scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -167,9 +163,26 @@ def build_graph(df, dataset):
     df = df.Define("goodMuons_pfRelIso04_all0", "Muon_pfRelIso04_all[goodMuons][0]")
 
     df = df.Define("goodCleanJets", "Jet_jetId >= 6 && (Jet_pt > 50 || Jet_puId >= 4) && Jet_pt > 30 && abs(Jet_eta) < 2.4 && wrem::cleanJetsFromLeptons(Jet_eta,Jet_phi,Muon_correctedEta[vetoMuons],Muon_correctedPhi[vetoMuons],Electron_eta[vetoElectrons],Electron_phi[vetoElectrons])")
-    df = df.Define("goodCleanJetsPt45", "goodCleanJets && Jet_pt > 45")
     df = df.Define("passIso", "goodMuons_pfRelIso04_all0 < 0.15")
 
+    ########################################################################
```

Comment:
Could some of this go inside the weights_and_corrs function? Or another one?

Reply:
It could, but that exists in theory_tools, so it is not the best to include experimental weights. We could create a new one, though

Reply:
Since Davide is also making changes that would affect this part in PR #122 let's keep it as it for now until this is worked out

Reply:
Ok, in general the define_weights_and_corrs function should probably go in another file, indeed. Doesn't have to be done now but should be eventually.